packages/auth/src/platform_browser/recaptcha/recaptcha_enterprise_verifier.ts

```diff
@@ -75,8 +75,8 @@ export class RecaptchaEnterpriseVerifier {
         if (auth.tenantId == null && RecaptchaEnterpriseVerifier.agentSiteKey != null) {
           return RecaptchaEnterpriseVerifier.agentSiteKey;
         }
-        if (auth.tenantId != null && RecaptchaEnterpriseVerifier.agentSiteKey !== undefined) {
-          return RecaptchaEnterpriseVerifier.siteKeys[auth.tenantId];
+        if (auth.tenantId != null && RecaptchaEnterpriseVerifier.tenantSiteKeys[auth.tenantId] !== undefined) {
```

Comment:
Run `yarn format` from the main package page (that will automatically wrap this line)